keylime/cloud_verifier_tornado.py

```diff
@@ -1198,7 +1212,10 @@ async def process_agent(agent, new_operational_state, failure=Failure(Component.
                 await process_agent(agent, states.FAILED, failure)
             else:
                 agent["operational_state"] = states.GET_QUOTE
-                cb = functools.partial(invoke_get_quote, agent, ima_policy, True, timeout=timeout)
+
+                def cb_quote_retry():
+                    asyncio.run(invoke_get_quote(agent, ima_policy, True, timeout=timeout))
+
```

Comment:
Former point: since `invoke_get_quote` takes 4 arguments, partial application with 4 parameters is equivalent to the complete `run` call, to the best of my understanding.

Latter point: I am not getting your point, can you please explain what makes the change not equivalent?

Reply:
@aplanas are you referring to the behavior with respect to async calls? I am struggling with it since I noticed my change breaks the code badly (e.g. integration tests are failing), maybe you have some suggestion based on your better understanding of Python's async and similar?